packages/ata/src/edgeCases.ts

```diff
@@ -57,7 +58,15 @@ export const mapModuleNameToModule = (moduleSpecifier: string) => {
     "zlib",
   ]
 
-  if (builtInNodeMods.includes(moduleSpecifier.replace("node:", ""))) {
+  const builtInNodeModsWithMandatoryPrefix = [
```

Comment:
Every new module node adds is going to be locked behind the node: scheme; I think it'd be better to hardcode the current list of modules that don't need the prefix, then always load of the prefix is present in an import.